src/methods/method_overloading/hard/q9/Overload9.java

```diff
@@ -0,0 +1,34 @@
+package methods.method_overloading.hard.q9;
+
+/**
+ * [J] Write a Java Program that passes in a user's first name and age and
```

Comment:
This question should be rewritten because we did not tell them the method name. Also, we might just be able to get rid of the main method in this one (but that is up to you). If you chose not to, we need to try and avoid the hardcoding though.

Reply:
We can also do the thing Bowen suggested though and give them the main method and just have them write the overloaded methods

Reply:
sadly I had already finished converting this one to input test before the meeting happened

Reply:
All good